core/src/main/java/com/novoda/landingstrip/LandingStrip.java

```diff
@@ -26,6 +29,7 @@
     private final ScrollOffsetCalculator scrollOffsetCalculator;
     private final FastForwarder fastForwarder;
 
+    private Set<ScrollingPageChangeListener> onScrollingPageChangeListeners;
```

Comment:
I'm not convinced on that a `list` is clearer to read than a `set`, but it's not a deal breaker in this circumstance, sure we can change that. I'm sure that if we add that `equals` we'll remember to also change the list to a set ;)